cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -221,6 +247,42 @@ mkTracers traceOptions tracer = Tracers
       traceWith (enableConsensusTracer Consensus.mempoolTracer
                 $ withName "Mempool" tracer) ev
 
+    teeForge
+      :: ForgeTracers
+      -> TracingFormatting
+      -> TracingVerbosity
+      -> Trace IO Text
+      -> Tracer IO (WithSeverity (Consensus.TraceForgeEvent blk (GenTx blk)))
+    teeForge ft tform tverb tr = Tracer $ \ev -> do
+      traceWith (teeForge' tr) ev
+      flip traceWith ev $ fanning $ \(WithSeverity _ e) ->
+        case e of
+          Consensus.TraceForgeEvent{}         -> teeForge' (ftForged ft)
```

Comment:
Alignment of `->`: From style doc: 
> Do not vertically align code. Mainly because the need to change identifiers and re-aligning the other lines causes semantically null changes in the diff.

Reply:
Done.